code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -567,3 +567,5 @@ Difficulty: Medium
 		playsound(C.loc,'sound/effects/hit_punch.ogg', 80, 1, 1)
 		C.spin(6,1)
 	..(targets, user, 60)
+
+/mob/living/simple_animal/hostile/megafauna/dragon/space_dragon/AltClickOn(atom/movable/A)
```

Comment:
>ready
>literally no code

Reply:
That's the point of it.  The dragon AltClickOn() is the swoop trigger for lesser ash drakes.  While this is technically "no code", it's defining the fact that the Alt Click function for the space dragon should do nothing.

Reply:
The reason why this even became a notable issue was because the alt click used to be a trigger for a custom-coded tail sweep, after the changes were made to switch it over to the pre-made spell, the alt-click function of the space dragon was removed, so it defaulted back to the base definition, which was the swoop.